ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/RedHat.yml

```diff
@@ -8,19 +8,23 @@ Build_Tool_Packages:
   - alsa-lib-devel
   - autoconf
   - bind-utils
+  - bison
+  - bzip2
   - cpio
   - cups-devel
   - elfutils-libelf-devel
-  - expat-devel
```

Comment:
I believe this is a requirement for compiling either git or gcc, unless you have already tested and proven its not required?

Reply:
I basically removed it as it wasn't in the CentOS playbook so presumable wasn't too critical. Have verified that the GIT_Source and gcc_48 roles both work without `expat-devel` being installed on a RHEL6 system so this should be safe ... If not I'll take the blame and we can reinstate it later ;-)